crates/cargo-contract/src/cmd/info.rs

```diff
@@ -55,75 +62,90 @@ pub struct InfoCommand {
     #[clap(name = "output-json", long)]
     output_json: bool,
     /// Display the contract's Wasm bytecode.
-    #[clap(name = "binary", long)]
+    #[clap(name = "binary", long, conflicts_with = "all")]
     binary: bool,
+    /// Display all contracts addresses
+    #[clap(name = "all", long)]
+    all: bool,
 }
 
 impl InfoCommand {
     pub fn run(&self) -> Result<(), ErrorVariant> {
-        tracing::debug!(
-            "Getting contract information for AccountId {:?}",
-            self.contract
-        );
-
         Runtime::new()?.block_on(async {
```

Comment:
Move this to `main.rs` like the other commands, and make this fn `async`

Reply:
Done